clients/privacy-center/common/geolocation.ts

```diff
@@ -11,9 +13,13 @@ const VALID_ISO_3166_LOCATION_REGEX = /^\w{2,3}(-\w{2,3})?$/;
 // (see https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/adding-cloudfront-headers.html#cloudfront-headers-viewer-location)
 const CLOUDFRONT_HEADER_COUNTRY = "cloudfront-viewer-country";
 const CLOUDFRONT_HEADER_REGION = "cloudfront-viewer-country-region";
+const CLOUDFRONT_VIEWER_ADDRESS = "cloudfront-viewer-address";
+const X_FORWARDED_FOR = "x-forwarded-for";
 export const LOCATION_HEADERS = [
   CLOUDFRONT_HEADER_COUNTRY,
   CLOUDFRONT_HEADER_REGION,
+  CLOUDFRONT_VIEWER_ADDRESS,
+  X_FORWARDED_FOR,
```

Comment:
These should be removed:
```suggestion
```

Including those headers in this array has an important effect: they are used to control how an upstream cache should vary it's cache keys, etc. It's hard to explain that succinctly here, but I don't think you quite understand that impact so let's play it safe 👍

Reply:
It does look like `Vary` headers are complex 😬 I'll remove for now